velox/common/process/ProcessBase.h

```diff
@@ -18,51 +18,36 @@
 
 #include <pthread.h>
 #include <sys/types.h>
+#include <cstdint>
 #include <string>
 #include <vector>
 
-namespace facebook {
-namespace velox {
-namespace process {
+namespace facebook::velox::process {
 
-/**
- * Current executable's name.
- */
+/// Current executable's name.
 std::string getAppName();
 
-/**
- * This machine'a name.
- */
+/// This machine'a name.
 std::string getHostName();
 
-/**
- * Process identifier.
- */
+/// Process identifier.
 pid_t getProcessId();
 
-/**
- * Current thread's identifier.
- */
+/// Current thread's identifier.
 pthread_t getThreadId();
 
-/**
- * Get current working directory.
- */
+/// Get current working directory.
 std::string getCurrentDirectory();
 
-/**
- * Returns elapsed CPU nanoseconds on the calling thread
- */
+/// Returns elapsed CPU nanoseconds on the calling thread
```

Comment:
Just for my illumination what's the difference in those comment types? is /// a doc-string type comment?

Reply:
Back in the day we had a discussion to converge on the type of comments we use, mostly for consistency, but seems like doxygen understands the three slashes as a special comment type as it generates documentation. 

https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md?plain=1#L119-L137

Reply:
Long story short, three slashes sort of represent documentation; two slashes are just regular comments.

Reply:
Thanks!